web/modules/custom/asset_release/src/Controller/AssetReleaseController.php

```diff
@@ -73,10 +99,7 @@ public function add(RdfInterface $rdf_entity) {
    *   The access result object.
    */
   public function createAssetReleaseAccess(RdfInterface $rdf_entity) {
-    $user = $this->currentUser();
-    $membership = Og::getMembership($user, $rdf_entity);
-    // @todo: Remove check for empty after ISAICP-2369 is in.
-    return (!empty($membership) && $membership->hasPermission('create asset_release rdf_entity')) ? AccessResult::allowed() : AccessResult::forbidden();
+    return $this->ogAccess->userAccessEntity('create', $this->createNewAssetRelease($rdf_entity), $this->currentUser());
```

Comment:
Well, the only think that this thing is actually taking care of is mapping the proper CRUD operation automatically without needing to hardcode it. Also, it's a simple creation of an entity so I don't think that it's that much of an issue.